api/src/permissions/utils/default-permission.ts

```diff
@@ -0,0 +1,9 @@
+import type { Permission } from '@directus/types';
+
+export const DEFUAULT_PERMISSION = {
```

Comment:
This looks to only be used in one place and a single test as that (`apply-query/filter/index.test.ts`)?